{{ cookiecutter.project_name }}/pyproject.toml

```diff
@@ -0,0 +1,100 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.mypy]
+warn_return_any = true
+warn_unused_configs = true
+
+[tool.poetry]
+authors = ["{{ cookiecutter.author_name }} <{{ cookiecutter.author_email }}>"]
+classifiers = [
+  "Development Status :: {{ cookiecutter.development_status }}",
+  "License :: OSI Approved :: Apache Software License",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: {{ cookiecutter.python_version }}",
+]
+description = "{{ cookiecutter.short_description }}"
+license = "Apache-2.0"
+maintainers = ["{{ cookiecutter.author_name }} <{{ cookiecutter.author_email }}>"]
+name = "{{ cookiecutter.project_slug }}"
+readme = "README.md"
+repository = "https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_name }}"
+version = "0.1.0"
+
+[tool.poetry.dependencies]
+python = ">={{ cookiecutter.python_version }}"
+
+[tool.poetry.group.code_quality]
+optional = true
+
+[tool.poetry.group.code_quality.dependencies]
+mypy = ">=1.10.0,<2.0.0"
+ruff = ">=0.4.10,<1.0.0"
+typos = ">=1.22.8,<2.0.0"
+
+[tool.poetry.group.docs]
+optional = true
+
+[tool.poetry.group.docs.dependencies]
+added-value = ">=0.24.0,<1.0.0"
+docutils = "0.20.1"
```

Comment:
I remember it breaking alot when I was testing in tesk.

Reply:
Hmmm, okay. Would of course be good to know what exactly broke, and what versions would be associated with it. Sooner or later the version will have some security vulnerability associated with it and then we need to update it. And then all the docs would break?? Not really good...

Reply:
create an issue for it for the time being?

Reply:
done, updated it, this seems to not break with this new proj!